README.md

```diff
@@ -370,6 +370,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [crz](https://github.com/dhruvrajvanshi/crz) - Functional programming library
  * [denetmen](https://github.com/izniburak/denetmen) - Useful micro validator / check library
  * [emoji.cr](https://github.com/veelenga/emoji.cr) - Emoji library
+ * [Fluence](https://github.com/crystallabs/fluence) - WYSIWYG wiki using markdown and Git
```

Comment:
Let's move it [Services and Apps](https://github.com/veelenga/awesome-crystal#services-and-apps)